app/src/main/java/org/fossasia/openevent/app/core/auth/login/LoginFragment.java

```diff
@@ -49,25 +49,50 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     @Override
     public void onStart() {
         super.onStart();
-        getPresenter().attach(this);
-        binding.setLogin(getPresenter().getLogin());
-        getPresenter().start();
+
+        //attaching the view model
+        loginFragmentViewModel = ViewModelProviders.of(this, viewModelFactory).get(LoginViewModel.class);
+        handleNewIntent();
+        handleEmail();
+        binding.setLogin(loginFragmentViewModel.getLogin());
         binding.btnLogin.setOnClickListener(view -> {
             if (!validator.validate())
                 return;
 
             ViewUtils.hideKeyboard(view);
-
             String url = binding.url.baseUrl.getText().toString().trim();
-            getPresenter().setBaseUrl(url, binding.url.overrideUrl.isChecked());
-            getPresenter().login();
+
+            loginFragmentViewModel.setBaseUrl(url, binding.url.overrideUrl.isChecked());
+            loginFragmentViewModel.showProgress().observe(this, this::showProgress);
```

Comment:
This should be unconditionally binded above

All LiveData should be unconditionally binded